test/signalrclient-e2e-tests/targetver.h

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
I don't think this file is needed at all (on the contrary - it can be harmful if we try making the tests run cross-plat)